src/server/config/schema.js

```diff
@@ -5,6 +5,16 @@ import os from 'os';
 import { fromRoot } from '../../utils';
 import { getData } from '../path';
 
+function createServerCorsSchema(defaultValue) {
+  return Joi
+    .alternatives()
+    .try([
+      Joi.boolean(),
+      Joi.object(),
```

Comment:
__question:__ is there any reason why we don't want to define all 7 properties that `cors` object accepts (https://hapijs.com/api/14.2.0)? Or maybe even just a subset of properties _we_ want to support. 

I know it requires more work and likely some documentation as well, but this feature seems to be important enough to not defer it to Hapi entirely. It will also make migration to the new platform a little bit easier as we'll know exactly what our ExpressJS server should support (including default values).

Reply:
I'll second this, though I'd upgrade it from a question to a bug.  These exposed configurations are a Kibana feature that happens to be powered by hapi, so we should have all of the relevant validations.